src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,70 @@
+$stopwatch-size: 80vmin;
+$stopwatch-center: $stopwatch-size / 2;
+$center-size: 5vmin;
+$center-center: $center-size / 2;
+$minutes-hand-width: 3vmin;
+$minutes-hand-heigh: 20vmin;
+$minutes-hand-x-center: $minutes-hand-width / 2;
+$seconds-hand-width: 1.5vmin;
+$seconds-hand-heigh: 38vmin;
+$seconds-hand-x-center: $seconds-hand-width / 2;
+
+.stopwatch {
+  width: $stopwatch-size;
+  height: $stopwatch-size;
+  position: relative;
+
+  border-radius: 50%;
+  border: 1vmin dotted #000;
+
+  &--speed-up {
+    --minute-duration: 10s;
+  }
+
+  &__center {
+    width: $center-size;
+    height: $center-size;
+    border-radius: 50%;
+    background-color: #f6a603;
+
+    position: absolute;
+    left: $stopwatch-center - $center-center;
+    bottom: $stopwatch-center - $center-center;
+  }
+
+  &__minutes-hand {
+    width: $minutes-hand-width;
+    height: $minutes-hand-heigh;
+    background-color: #0700ff;
+
+    position: absolute;
+    bottom: $stopwatch-center;
+    left: $stopwatch-center - $minutes-hand-x-center;
+
+    animation: rotate calc(var(--minute-duration) * 60) steps(60) infinite;
```

Comment:
The animation timing function `steps(60)` might not be appropriate for a smooth rotation. Consider using `linear` for a continuous rotation effect.